src/components/Facility/DischargedPatientsList.tsx

```diff
@@ -136,6 +213,26 @@ const DischargedPatientsList = ({
     }
   }, [qParams]);
 
+  const dateRangeFields = [
+    [params.created_date_before, params.created_date_after],
+    [params.modified_date_before, params.modified_date_after],
+    [params.date_declared_positive_before, params.date_declared_positive_after],
+    [params.last_vaccinated_date_before, params.last_vaccinated_date_after],
+    [
+      params.last_consultation_encounter_date_before,
+      params.last_consultation_encounter_date_after,
+    ],
+    [
+      params.last_consultation_discharge_date_before,
+      params.last_consultation_discharge_date_after,
+    ],
+  ];
+
+  const isExportAllowed = dateRangeFields.some(([startDate, endDate]) => {
+    const days = calculateDaysBetween(startDate, endDate);
+    return days >= 0 && days <= 7 && days !== 0;
```

Comment:
why check for greater than equal to zero and then for not equal to zero here? 🤔  Greater than zero alone should suffice.